### ISSUE TITLE
In v1.6.19, "-N" no longer forces protocol to be an integer

### ISSUE THREAD

**Issue by philhagen** [phaag/nfdump#213]
In 1.6.18 and prior, the `-N` flag would force all numbers to be raw and would also display the protocol as its integer value.  In 1.6.19, this is no longer the case:

```
$ nfdump -V
nfdump: Version: NSEL-NEL1.6.18
$ nfdump -q -N -R . -o 'fmt:%pr' | head -n 1
6
```

```
$ nfdump -V
nfdump: Version: NSEL-NEL1.6.19
$ nfdump -q -N -R . -o 'fmt:%pr' | head -n 1
TCP
```

Just hoping to confirm if this is the new standard or if this was mistaken and will be reverted in the future.  This significantly affects several post-processing scripts we use and I want to know if those should be ported forward, or if we should stick with 1.6.18 for a while.

**Comment by phaag** [comment]
Ohh - that's for sure a bug! This has to be reverted to the old standard!

**Comment by philhagen** [comment]
ok good to know.  I'll version-lock us to 1.6.18 for now

**Comment by phaag** [comment]
Fixed! 1.6.20 published!
